apps/functions-subscription/src/adapters/azure/cosmosdb/__tests__/trial.test.ts

```diff
@@ -84,6 +84,82 @@ describe('makeTrialsCosmosContainer', () => {
       expect(mockDB.container('').item).toBeCalledWith(id, ownerId);
     });
   });
+  describe('list', () => {
+    const anotherTrial = {
+      ...aTrial,
+      id: 'anotherTrialId' as TrialId,
+    };
+
+    it('should return list of trials', async () => {
+      const mockDB = makeDatabaseMock();
+
+      const trials = [aTrial, anotherTrial];
+
+      mockDB.container('').items.query.mockReturnValueOnce({
+        fetchAll: () => Promise.resolve({ resources: trials }),
+      });
+
+      const actual = await makeTrialsCosmosContainer(
+        mockDB as unknown as Database,
+        containerName,
+      ).list({ pageSize: 25 })();
+
+      expect(actual).toStrictEqual(E.right(trials));
+      expect(mockDB.container('').items.query).toHaveBeenCalledTimes(1);
+    });
+
+    it('should return list of trials with id greater than "a"', async () => {
```

Comment:
**suggestion**: following a previous suggestion, we can rename the test something like

`should make the query with pageSize (or the name used in the query) and minimumId)`

what do you think?

If you agree, I'd be consistent with other unit tests as well

Reply:
Updated in 3dc0bb86816f2a2f2f3fbcf3807f68fc3e45837a